packages/frontend-shared/src/gql-components/StatusBadge.vue

```diff
@@ -12,6 +12,7 @@
           />
           {{ titleOn }}
           <i-cy-chevron-down-small_x8
+            v-if="!props.isRunning"
```

Comment:
Would it make more sense for this to be:

```suggestion
            v-if="menuItems.length > 0"
```

Functionally the same today but might future proof things a bit.

Reply:
Better yet, it's not even a list of items anymore, so we can just

```html
<MenuItems v-if="!props.isRunning">
  <MenuItem>
    <!-- stuff -->
```

No need to use a list when you either have 1 item or 0 items.